mlrun/runtimes/kubejob.py

```diff
@@ -53,6 +54,20 @@ def is_deployed(self):
             return True
         return False
 
+    def from_source_archive(self, source, pythonpath=None, at_runtime=True):
```

Comment:
Now I understand that the previous comments regarding was irrelevant, and this method can be used to both loading on runtime or build time, so the doc there should be changed.
usually `from_x` methods are constructor, which is not the case, how about `with_code_from_archive` ?

Reply:
Also maybe `pull_on_runtime` ?
I'm considering whether this should be a bool, or a mode to leave room for more than 2 options, for example maybe we'll allow the archive to be given at "configuration" time (directly from the notebook) when calling this function

Reply:
done